app/views/customers/_form.html.slim

```diff
@@ -55,32 +56,45 @@
               = image_tag 'powerdBy.png', height: '35', width: '125'
             h4 Sendaguy now uses Stripe, a leader in online and mobile payments, for all financial transactions.
         .next-btn-wrapper
-          = link_to '#', class: 'hidden', id: 'next-btn' do
+          = link_to '#', class: 'hidden next-btn' do
             = fa_icon 'chevron-right'
     h2 Summary & Payment
     section 
       .content-wrapper.signup-form
         .back-btn-wrapper
-          = link_to '#', class: 'mb20', data: { back: 'card-details' }, id: 'back-btn' do
+          = link_to '#', class: 'mb20 back-btn' do
             = fa_icon 'chevron-left'
         .col-md-6.center-panel.text-center.mt20
-          h3.text-center Enter your details for signup
+          h3.text-center.signup-header
+            span Enter your details for signup
+            strong.ml10 OR
+            span.ml10.w30-pct
+              = link_to 'Login', '#', class: 'service-request-login-link btn btn-danger btn-danger btn-lg'
+          h3.text-center.signin-header.hidden
+            span Enter your details for signin
+            strong.ml10 OR
+            span.ml10
+              = link_to 'Sign Up', '#', class: 'service-request-logout-link btn btn-danger btn-lg'
           .row.mt40
             .col-sm-6.mt10
               .form-inputs.text-left
-                = f.input :name, required: true, autofocus: true, placeholder: 'Your Name', label: false
-                = f.input :customer_account_name, required: true, placeholder: 'Your Business Name', label: false
-                = f.input :email, as: :email, required: true, placeholder: 'Email', label: false
-                = f.input :unconfirmed_phone, as: :tel, placeholder: 'Phone Number', label: false, input_html: { class: 'us_phone_number' }, validate: true
-                = f.input :password, required: true, placeholder: 'Your Password', label: false
-                = f.input :password_confirmation, required: true, placeholder: 'Confirm Password', label: false
-                = f.input :tos_accepted, as: :boolean, label: tos_label
+                .sign-up-fields
+                  = f.input :name, required: true, autofocus: true, placeholder: 'Your Name', label: false
+                  = f.input :customer_account_name, required: true, placeholder: 'Your Business Name', label: false
+                  = f.input :email, as: :email, required: true, placeholder: 'Email', label: false
+                  = f.input :unconfirmed_phone, as: :tel, placeholder: 'Phone Number', label: false, input_html: { class: 'us_phone_number' }, validate: true
+                  = f.input :password, required: true, placeholder: 'Your Password', label: false
+                  = f.input :password_confirmation, required: true, placeholder: 'Confirm Password', label: false
+                  = f.input :tos_accepted, as: :boolean, label: tos_label
+                .sign-in-fields.hidden
```

Comment:
I'm not sure if it's correct way to use one form for SignUp and for SignIn functionality. I think it will be better to separate it to different forms, please describe idea of this choice if you think it's justified.